lib/assert.js

```diff
@@ -20,6 +24,20 @@ function test(ok, opts) {
 	}
 }
 
+function enhanceAssert(assert) {
+	return empower(assert,
+		powerAssertFormatter({
+			renderers: [
+				powerAssertRenderers.AssertionRenderer,
+				powerAssertRenderers.SuccinctRenderer
+			]
+		}),
+		{
+			destructive: true
+		}
+	);
```

Comment:
Yeah, I don't see why we wouldn't want to decorate all of them, even though `t.ok`/`notOk` are the ones with the most gain.

> (Is t.assert public API?)

No, it's only there for legacy reasons.